Lib/enum.py

```diff
@@ -946,3 +942,42 @@ def _decompose(flag, value):
         # we have the breakdown, don't need the value member itself
         members.pop(0)
     return members, not_covered
+
+def global_int_repr(self):
+    return '%s.%s' % (self.__class__.__module__, self.name)
```

Comment:
What if add an attribute to the enum class which is set to the class name by default and can be set to the module name for enums whose members are exposed as globals?
```python
    return '%s.%s' % (self.__class__._namespacename, self.name)
```
It will make easier customization of the repr.

Reply:
I think writing a custom `__repr__` is not that difficult.